dpgen/generator/run.py

```diff
@@ -62,6 +62,11 @@
 from dpgen import ROOT_PATH
 from pymatgen.io.vasp import Incar,Kpoints,Potcar
 from dpgen.auto_test.lib.vasp import make_kspacing_kpoints
+try:
+    from gromacs.fileformats.mdp import MDP
+except ImportError:
+    dlog.info("GromacsWrapper>=0.8.0 is needed for DP-GEN + Gromacs.")
```

Comment:
This information should not appear here since not everyone will install `gromacs` package.

Reply:
I  added GromacsWrapper dependency in setup.py, do you think it's appropriate? 
Same problems may occur on openbabel, rdkit and other softwares needed to support other engines, such as Gromacs, Amber, VASP, etc. @amcadmus  What do you think?